packages/api/lambdas/cleanExecutions.js

```diff
@@ -2,138 +2,143 @@
 
 'use strict';
 
-const { getEsClient, esConfig } = require('@cumulus/es-client/search');
-const moment = require('moment');
+/**
+ * This lambda has been commented out because it needs to be rewritten for PG instead of running
+ * in ElasticSearch. This will be done in CUMULUS-XXXX
+ */
+
+// const { getEsClient, esConfig } = require('@cumulus/es-client/search');
+// const moment = require('moment');
 const Logger = require('@cumulus/logger');
-const { sleep } = require('@cumulus/common');
+// const { sleep } = require('@cumulus/common');
```

Comment:
Remove comments

Reply:
We should avoid having commented-out code generally. We can rely on the git diff if we need to reference what was here.

Reply:
hah. Well, I completely removed the lambda and test, and @jennyhliu suggested that I add them back and comment out except for a dummy handler. So I'm open to suggestions on why one method is better than the other...

Reply:
(see comment at top of file)

Reply:
If we remove lambda and test here, we have to remove the related tf code, that's why I suggested to have dummy handler since we want to keep the lambda with RDS implementation.
Now I think we can delete the code and leave only the dummy handler.

Reply:
Ok, I can remove the commented code, if everyone is in agreement there. 
What about the test file, should I remove the file altogether? Or if not, what should I leave in the file if not commented out tests?

Reply:
Sounds good. Removing all commented code is certainly the preference as it's all in git if we need to get back to it.

Reply:
code removed but dummy handler left in place